src/components/ClientCard/ClientCard.jsx

```diff
@@ -20,24 +20,20 @@ const ClientCard = props => (
   </li>
 );
 
-const mapStateToProps = state => ({
-
-});
-
 
 const mapDispatchToProps = dispatch => ({
-  setActiveChat: chat => dispatch(setActiveChat(chat)),
+  setActiveChat: chatId => dispatch(setActiveChat(chatId)),
 });
 
 
 export default connect(
-  mapStateToProps,
+  () => ({}),
```

Comment:
I looked around for an option. I think this might be the best case to appease the linter and keep readability:

<img width="520" alt="screenshot 2017-03-17 11 34 56" src="https://cloud.githubusercontent.com/assets/12987958/24050632/d0a84a20-0b05-11e7-93eb-6812d096b21f.png">